docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: "3.9"
```

Comment:
This docker-compose is intended only for local development, right?

I'd recommend adding 2 more volumes (named volumes, not the directory ones), one for media (/code/media) and one for the database.

Reply:
There's already volume for /code, so why another for /code/media?